src/js/apps/forms/widgets/widgets_header_app.js

```diff
@@ -8,13 +7,10 @@ import { FormWidgetsHeaderView } from 'js/views/forms/form/widgets/widget_header
 export default App.extend({
   beforeStart({ patient, form }) {
     const workspacePatient = Radio.request('entities', 'fetch:workspacePatients:byPatient', patient.id);
-    const fields = map(form.getWidgetFields(), fieldName => {
-      return Radio.request('entities', 'fetch:patientFields:model', patient.id, fieldName);
```

Comment:
we should check about `fetch:patientFields:model`

Reply:
Yep, removed 👍